assets/js/components/dashboard-splash/DashboardSplashApp.js

```diff
@@ -23,16 +23,16 @@ import '../publisher-wins';
 import Data from 'googlesitekit-data';
 import { STORE_NAME as CORE_SITE } from '../../googlesitekit/datastore/site/constants';
 import SetupUsingProxy from '../setup/SetupUsingProxy';
-import LegacyDashboardSplashApp from './LegacyDashboardSplashApp';
+import SetupUsingGCP from '../legacy-setup/SetupUsingGCP';
 const { useSelect } = Data;
 
 export default function DashboardSplashApp() {
 	const usingProxy = useSelect( ( select ) => select( CORE_SITE ).isUsingProxy() );
 
 	if ( usingProxy === true ) {
 		return <SetupUsingProxy />;
-	} else if ( usingProxy === false ) {
-		return <LegacyDashboardSplashApp />;
+	} if ( usingProxy === false ) {
```

Comment:
Suuuper minor, but let's use a newline here.

```suggestion
	}
	
	if ( usingProxy === false ) {
```